### ISSUE TITLE
Project not working on iOS Devices (iphone 8 and later [with iOS 8 & above])

### ISSUE THREAD

**Issue by rohats871** [avinashkranjan/Friday#61]
My Xcode is all updated to iOS 9 and Above  plus i tried on iOS 8.
<img width="1176" alt="Screenshot 2021-03-10 at 3 45 21 PM" src="https://user-images.githubusercontent.com/68010359/110613742-a14b5880-81b7-11eb-94a3-f0fb538c7834.png">
 

warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Runner' from project 'Runner')
    warning: Capabilities for Signing & Capabilities may not function correctly because its entitlements use a placeholder team ID. To resolve this, select a development team in the Runner editor. (in target 'Runner' from project 'Runner')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Pods-Runner' from project 'Pods')
.
.
.
(This was done on simulator)

**Comment by avinashkranjan** [comment]
I haven't checked it any iOS devices or Simulator..But If there's this issue I'll check it once.. But if you're looking forward to work on this bug/issue kindly inform me. I'll assign it to you..

**Comment by rohats871** [comment]
I would love to do that , but I have already tried it so I might require someone else also

**Comment by avinashkranjan** [comment]
Okay, I'll check and I'll let you know

**Comment by rohats871** [comment]
@avinashkranjan  assign me this let me try again on my end .

**Comment by rohats871** [comment]
@avinashkranjan i might require the ios version of the GoogleServices file

**Comment by rohats871** [comment]
> Hey @rohats871 Sending you ios version of the GoogleServices file on Discord, Kindly Ping me on Discord for the same

can i have it here please?

**Comment by avinashkranjan** [comment]
@rohats871 That file is not supported here..

**Comment by avinashkranjan** [comment]
Any Updates @rohats871 ?
